scikits/fitting/delaunay.py

```diff
@@ -1,70 +1,58 @@
-try:
-    # pylint: disable-msg=E0611
-    import scikits.delaunay as delaunay
-    delaunay_found = True
-except ImportError:
-    # In scipy 0.6.0 and before, the delaunay module is in the sandbox
-    try:
-        # pylint: disable-msg=E0611,F0401
-        import scipy.sandbox.delaunay as delaunay
-        delaunay_found = True
-    except ImportError:
-        # Matplotlib also has delaunay module these days - use as last resort (more convenient than scikits)
-        try:
-            # pylint: disable-msg=E0611,F0401
-            import matplotlib.delaunay as delaunay
-            delaunay_found = True
-        except ImportError:
-            delaunay_found = False
-
-from .generic import ScatterFit, GridFit, NotFittedError
+# Interpolators based on triangulation available since scipy 0.9.0
+from scipy.interpolate import (CloughTocher2DInterpolator, LinearNDInterpolator,
+                               NearestNDInterpolator)
+
+from .generic import ScatterFit, NotFittedError
+
+
+logger = logging.getLogger(__name__)
 
 # ----------------------------------------------------------------------------------------------------------------------
 # --- CLASS :  Delaunay2DScatterFit
 # ----------------------------------------------------------------------------------------------------------------------
 
 
 class Delaunay2DScatterFit(ScatterFit):
-    """Interpolate scalar function of 2-D data, based on Delaunay triangulation
-    (scattered data version).
+    """Interpolate scalar function of 2-D data, based on Delaunay triangulation.
 
     The *x* data for this object should have two rows, containing the 'x' and
     'y' coordinates of points in a plane. The 2-D points are therefore stored as
     column vectors in *x*. The *y* data for this object is a 1-D array, which
     represents the scalar 'z' value of the function defined on the plane
-    (the symbols in quotation marks are the names for these variables used in
-    the ``delaunay`` documentation). The 2-D *x* coordinates do not have to
-    lie on a regular grid, and can be in any order. Jittering a regular grid
-    seems to be troublesome, though...
+    (the symbols in quotation marks are the typical mathematical names for
+    these variables). The 2-D *x* coordinates do not have to lie on a regular
+    grid, and can be in any order.
 
     Parameters
     ----------
-    interp_type : {'nn'}, optional
-        String indicating type of interpolation (only 'nn' currently supported)
+    interp_type : {'cubic', 'linear', 'nearest'}, optional
+        String indicating type of interpolation
     default_val : float, optional
         Default value used when trying to extrapolate beyond convex hull of
-        known data
+        known data (ignored for 'nearest')
     jitter : bool, optional
         True to add small amount of jitter to *x* to make degenerate
-        triangulation unlikely
+        triangulation unlikely (generally not needed with Qhull back-end)
```

Comment:
Is the comment about Qhull still relevant now that you have only the scipy backend?